xtask/src/check_feature_dependencies.rs

```diff
@@ -63,6 +61,76 @@ pub fn check_feature_dependencies(shell: Shell, arguments: Arguments) -> anyhow:
             default_features: false,
             search_terms: &[Search::Negative("naga")],
         },
+        Requirement {
```

Comment:
I didn't know we have those tests. Absolute gold

Reply:
I'm missing a test for `webgl` being enabled not enabling `glow` on a desktop platform since this is one of the failure modes we used to have.

Reply:
Added.